include/cuco/static_map.cuh

```diff
@@ -276,6 +276,33 @@ class static_map {
                 Hash hash          = Hash{},
                 KeyEqual key_equal = KeyEqual{});
 
+  /**
+   * @brief Indicates whether the keys in the range `[first, last)` are missing from the map.
+   *
+   * Writes a `bool` to `(output + i)` indicating if the key `*(first + i)` does not exist in the map.
+   *
+   * @tparam InputIt Device accessible input iterator whose `value_type` is
+   * convertible to the map's `key_type`
+   * @tparam OutputIt Device accessible output iterator whose `value_type` is
+   * convertible to the map's `mapped_type`
+   * @tparam Hash Unary callable type
+   * @tparam KeyEqual Binary callable type
+   * @param first Beginning of the sequence of keys
+   * @param last End of the sequence of keys
+   * @param output_begin Beginning of the sequence of booleans for the presence of each key
```

Comment:
Strictly speaking, the output sequence doesn't need to literally be bools, it just needs to be assignable from bools. 
```suggestion
   * @param output_begin Beginning of the output sequence to which a `bool` is assigned indicating the present of each key 
```